frontend/src/utils/replace_content_with_refs.ts

```
@@ -105,21 +110,37 @@ function replace_nested_nodes(
+// better.
+function give_new_ids(curr_node: DBDetails): void {
+  const que: ColumnSchema[] = [...curr_node.schema];
+  let current_q_idx: number = 0;
+  const used_ids: Set<string> = new Set();
+
+  while (current_q_idx < que.length) {
+    if (used_ids.has(que[current_q_idx].id)) {
+      que[current_q_idx].id = uuidv4();
+
+      if (que[current_q_idx].children) {
+        for (const child of que[current_q_idx].children!) {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)